batik-util/src/main/java/org/apache/batik/util/CleanerThread.java

```diff
@@ -99,25 +102,30 @@ protected CleanerThread() {
     }
 
     public void run() {
-        while(true) {
-            try {
-                Reference ref;
-                try {
-                    ref = queue.remove();
+        try {
+            while(!shutdownLatch.await(100, TimeUnit.MILLISECONDS)) {
```

Comment:
Do you have a reasoning as to why 100ms?

Reply:
Not against a timeout, but 100ms looks way too small.

Reply:
Changed it to seconds.